.github/templates/tilgangsstyring-pnd.md

```diff
@@ -66,3 +66,49 @@ labels: 'kind/deploy_patch, team/tilgangsstyring'
 [^1]: Approve pending prod releases by clicking the blue production chip and clicking approve. ![image-20240306133832594](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-pending-approval-screen.png)
 [^2]: Look for any blue production chips. ![image-20240306133137061](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-prod-button.png)
 [^3]: Look for any blue TT02 chips. ![tt02-button](https://raw.githubusercontent.com/Altinn/altinn-authorization/main/.github/images/ado-tt02-button.png)
+
+---
+
+## Deployrutine
+Det som skal deployes til PROD skal ha vært ute i TT02 i en uke. Før man deployer må man sørge for at man ikke introduserer nye feil, så da må man sjekke et par steder:
+
+### Sjekk Slack-kanalene
+
+<img width="481" alt="Pasted image 20231115133133" src="https://github.com/Altinn/altinn-authorization/assets/59682632/b2e42023-6e37-4252-be49-cc0432c752ed">
```

Comment:
I would try to keep the markdown free from HTML if at all possible, and I'd argue that it's easily possible in this case

Reply:
Also - where are these images hosted? How permanent are the links? You should probably just add them to the images folder like the rest of the images.

Reply:
Removed html. 

I don't know where the files are hosted, they were added automatically after drag-dropping images into the editor... Copied the screenshots to the images folder instead.